### PR TITLE
Update cy.yml

### PR DESCRIPTION
**Pull request by kerryalyons**
Hint content needed on page to be clear to grantee what the link is between the question and asking grantees to upload a risk register.

## Proposed changes

This is the Welsh version of the content on the English page. New hint content added.

## Further comments

Devs to review and check content design have added hint content correctly, and in the right place.

## Screenshot (if applicable)

image

## Checklist

- [ ] Added tests for new functionality, or updated existing tests
- [ ] Updated `README.md` if necessary

### CODE REVIEW

**Comment by kerryalyons** [comment]
Added new hint content to How will your project be managed?

File: config/locales/funding_application/gp_open_medium/managed/cy.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ cy:
   managed:
     page_title: "Sut bydd eich prosiect yn cael ei reoli?"
     page_heading: "Sut bydd eich prosiect yn cael ei reoli?"
+    page_hint: "Dywedwch wrthym pwy fydd yn gwneud penderfyniadau, profiad y bobl dan sylw a'u rolau yn y prosiect. Rydym yn gwybod bod gan bob prosiect risgiau. Rydym am wybod pa risgiau sydd wedi'u hystyried a sut y cânt eu rheoli.
```
Comment (ptrelease) [inline config/locales/funding_application/gp_open_medium/managed/cy.yml:5]: @kerryalyons this is still missing the double-dangler at the end, so tests still fail.

Comment (kerryalyons) [reply config/locales/funding_application/gp_open_medium/managed/cy.yml]: Added. Should be ok now.

**Comment by ptrelease** [comment]
Closing and creating a new branch with changes, as medium journey did not work with this branch.

**Pull request closed without merging**